metrics/nodehealthcheck.go

```diff
@@ -32,15 +32,42 @@ var (
 	)
 )
 
+var (
+	// NodeHealthCheckUnhealthyNodeOngoingRemediation is a Prometheus metric, which reports an ongoing remediation of a node
+	NodeHealthCheckUnhealthyNodeOngoingRemediation = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "nodehealthcheck_unhealthy_node_ongoing_remediation",
+			Help: "Indication of an ongoing remediation of an unhealthy node",
+		}, []string{"name", "namespace", "remediation"},
+	)
+)
+
 func InitializeNodeHealthCheckMetrics() {
 	metrics.Registry.MustRegister(
 		NodeHealthCheckOldRemediationCR,
+		NodeHealthCheckUnhealthyNodeOngoingRemediation,
 	)
 }
 
-func ObserveNodeHealthCheckOldRemediationCR(name string, namespace string) {
+func ObserveNodeHealthCheckOldRemediationCR(name, namespace string) {
 	NodeHealthCheckOldRemediationCR.With(prometheus.Labels{
 		"name":      name,
 		"namespace": namespace,
 	}).Inc()
 }
+
+func TrackNodeHealthCheckUnhealthyNodeOngoingRemediation(name, namespace, remediation string) {
```

Comment:
It's best practice to prefix these functions with `Observe...`
What about `ObserveNodeHealthCheckRemediationCreated(...)` and `ObserveNodeHealthCheckRemediationDeleted(...)`?

Reply:
I'm fine with that, what do you think @clobrano?